roles/container-engine/cri-o/defaults/main.yml

```diff
@@ -73,6 +73,13 @@ crun_runtime:
   type: oci
   root: /run/crun
 
+# youki is an implementation of the OCI runtime-spec in Rust, similar to runc.
+youki_runtime:
+  name: youki
+  path: /usr/bin/youki
```

Comment:
for consistency this should use either `{{ youki_bin_dir }}` or the global `{{ bin_dir }}`